docs/configuration.md

```diff
@@ -18,6 +18,11 @@ template:
     # Name of the index file (default is "README.md")
     index: index.md
 
+# List of directories to include in the processing (default is all markdown files)
+# Paths are relative to the repository root
```

Comment:
Could you maybe make it explicit that it's empty by default (if empty then it's "all markdown files")